### DISCUSSION TITLE
中微子大作业JUNO问题汇总

### DISCUSSION THREAD

**Discussion by Bababeluma** [physics-data/faq#79] [Q&A] [answered]
请问中微子大作业是要追踪每根光线模拟整个物理过程，还是只要能生成一个以假乱真符合统计规律的data就行？

**Comment by Harry-Chen** [comment]
@Berrysoft

**Comment by Berrysoft** [comment, marked as answer]
都可以。既可以对每个光子计算其光学过程，也可以先构建Probe直接做泊松过程采样。采取哪种模拟思路应当在实验报告中有体现，因为这是算法重点。

**Reply by Berrysoft** [reply]
如果先构建Probe，即非齐次泊松过程的期望，应当给出Probe构建的方法。

**Reply by Bababeluma** [reply]
好的，助教辛苦了！

**Comment by Even-Tang** [comment]
@Berrysoft 
’‘’
当你计算得到波形之后，应该在0到1000ns每1ns采样一个数据点，作为波形的输出结果。这是因为 PMT 输出的并不是模拟信号，而是电子采样之后的信号（联想电视的模拟信号与数字信号）。
电子学噪声可以视作对一个周期无理（或者不是很有理）的正弦函数的采样。要求周期无理是因为看起来随机，要求正弦函数是因为要求很大的区域内积分趋近于0。如果你有别的好办法也可以，并在实验报告内说明。
‘’‘
这段采样信号是整数是什么意思，是把信号随便乘一个常数在各个时刻取整吗？噪声相对信号又有多大？
另外每个光子打到探测器的信号是一样的，那么采样到的信号只有时间先后的区别？除非一次事件中同一个PMT产生两个光子

**Reply by Berrysoft** [reply]
> 这段采样信号是整数是什么意思

这段没说采样信号是整数，但是时刻是整数。

> 噪声相对信号又有多大？

噪声可以取一个你认为合理的幅值。

> 采样到的信号只有时间先后的区别？

在这个简化模型下，是的。

**Reply by Even-Tang** [reply]
谢谢助教， 采样信号是整数是因为输出格式是‘<i2'

**Reply by Berrysoft** [reply]
算出来之后取整就好，没有给出的常数可以取自己认为合理的
